doc/architecture/layers.sql

```diff
@@ -0,0 +1,112 @@
+-- User table
+CREATE TABLE users (
+    ...
+);
+
+-- Project table
+CREATE TABLE map (
```

Comment:
As discussed in #245 we will have an additional "versions" table between the map and the individual layers. The layers should therefore reference this "versions" table instead of "map".